StantUiIosLibrary/Classes/Utils/UIImageViewExtension.swift

```diff
@@ -9,19 +9,31 @@ import UIKit
 
 extension UIImageView {
     
-    func showRounded(image: String, radius: CGFloat = 8) {
+    func showRounded(image: String, radius: CGFloat = 8, activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView()) {
         self.layer.cornerRadius  = radius
         self.layer.masksToBounds = true
         
-        self.show(image: image)
+        self.show(image: image, activityIndicator: activityIndicator)
     }
     
-    func show(image: String) {
-        if let url = NSURL(string: image) {
-            if let data = NSData(contentsOf: url as URL) {
-                self.image = UIImage(data: data as Data)
+    func show(image: String, activityIndicator: UIActivityIndicatorView) {
```

Comment:
Maybe the `image` name is not given the correct impression. Is it an `image_path`?

Reply:
Yes. It is, I'm refactoring it.